README.md

```diff
@@ -8,19 +13,20 @@ This is a library to serialize PHP variables in JSON format. It is similar of th
 but the output is a string JSON encoded. You can also unserialize the JSON generated by this tool and have you
 PHP content back.
 
-Supported features:
+**Supported features**:
 - Encode/Decode of scalar, null, array
-- Encode/Decode of objects
+- Encode/Decode of objects, including all the [`DateTime family objects`](http://php.net/manual/en/book.datetime.php)  (except `DatePeriod`).
```

Comment:
I tried serializing it and after checking out the results i found out it needs a very specific and concrete serialization strategy on its own.